parser/csv_tree_parser.go

```diff
@@ -34,6 +34,7 @@ func (s CSVTreeParser) ParseString(in string) (*treemap.Tree, error) {
 func parseNodes(in string) ([]treemap.Node, error) {
 	var nodes []treemap.Node
 	r := csv.NewReader(strings.NewReader(in))
+	r.LazyQuotes = true
 	for {
 		record, err := r.Read()
 		if err == io.EOF {
```

Comment:
Code coverage decreased because the following error check never triggers, as double-quotes are allowed here.

Any other error in the CSV reader we can trigger in the tests?

Reply:
looks ok!